odoo/addons/base/models/res_currency.py

```diff
@@ -365,10 +365,12 @@ def _sanitize_vals(self, vals):
         return vals
 
     def write(self, vals):
+        self.env['res.currency'].invalidate_model(['rate'])
```

Comment:
Hello, 
I don't understand. The invalidation should be done by the ORM because of `@api.depends('rate_ids.rate')`, no ?
Can you create a test for that?